tools/openblas_support.py

```diff
@@ -13,8 +13,8 @@
 from urllib.request import urlopen, Request
 from urllib.error import HTTPError
 
-OPENBLAS_V = '0.3.18'
-OPENBLAS_LONG = 'v0.3.18'
+OPENBLAS_V = '0.3.19'
+OPENBLAS_LONG = 'v0.3.19'
```

Comment:
If you'd like to have the new version available per the usual process, you could probably just copy Matti's most recent PR to the upstream repo and bump the version (3 lines to change):

https://github.com/MacPython/openblas-libs/pull/67

Since there have been no other PRs merged to that repo since Matti made that PR, you may get lucky and things could "just work." Then, after we merge that, and the binaries are available (*after* the CI runs on a merge proper), can come back to this PR.

Reply:
Proposed (and merged, several months ago) as MacPython/openblas-libs#69